entry_tasks/16_Pick_movie_by _genre_or_actor/main.py

```diff
@@ -0,0 +1,81 @@
+GENRES = {
+    "comedy": ["Meet the Parents", "Anger Management"],
+    "adventures": ["Mummy"],
+    "romantic": ["Vanilla Sky", "Meet Joe Black"],
+    "drama": ["Meet Joe Black"],
+    "thriller": ["Vanilla Sky"],
+    "action": ["Mission Impossible"],
+}
+CAST = {
+    'Meet the Parents': ['Robert De Niro', 'Ben Stiller'],
+    'Anger Management': ['Adam Sandler', 'Jack Nicholson'],
+    'Mummy': ['Brendan Fraser', 'Rachel Weisz'],
+    'Vanilla Sky': ['Tom Cruise', 'Penelope Cruz', 'Cameron Diaz'],
+    'Meet Joe Black': ['Brad Pitt', 'Anthony Hopkins'],
+    'Mission Impossible': ['Tom Cruise', 'Jeremy Renner']
+}
+
+cast_values = sum(CAST.values(), [])
```

Comment:
it make sense to move out main some constants, but not the result of some filtration, what you gonna do if next task would be also to update CAST by admin user input?

Reply:
Great that you find such solutions but in real world problem, structure code as simple as possible and readable.

For example:
```py
actors = {}

for cast in CAST.values():
    for actors in cast:
        actors.add(actor)

# or the same result
actors = {actor for cast in CAST.values() for actor in cast}
```
* google `unpack list of lists python`